cgroups/container.go

```diff
@@ -0,0 +1,60 @@
+package cgroups
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+)
+
+//-------------------------------------------------------------------------------------------------
```

Comment:
These comments are not very helpful. It's common to have godoc comments on exported types and methods https://go.dev/blog/godoc